frontend/public/components/monitoring/alerting.tsx

```diff
@@ -843,10 +897,65 @@ export const AlertRulesDetailsPage = withFallback(
       return `${nameLabel}{${_.map(otherLabels, (v, k) => `${k}="${v}"`).join(',')}}`;
     };
 
+    const ActiveAlerts = ({ activeAlerts, ruleID, activeNamespace }) => (
```

Comment:
I'd prefer to leave `ActiveAlerts` as a separate component. It was originally broken out to keep `AlertRulesDetailsPage` from becoming too large.